nomenclature/codelist.py

```diff
@@ -280,34 +280,70 @@ def to_yaml(self, path=None):
         else:
             return stream
 
-    def to_excel(self, excel_writer, sheet_name="definitions"):
-        """Write the codelist to an Excel spreadsheet
+    def to_pandas(self, sorted=False):
+        """Export the CodeList to a :class:`pandas.DataFrame`
 
         Parameters
         ----------
-        excel_writer : str, path or ExcelWriter object
-            Any valid string path, :class:`pathlib.Path` or :class:`pandas.ExcelWriter`
-        sheet_name : str
-            Name of sheet that will contain the codelist
+        sorted : bool, optional
+            Sort the codelist before exporting to csv.
         """
-
-        # open a new ExcelWriter instance (if necessary)
-        close = False
-        if not isinstance(excel_writer, pd.ExcelWriter):
-            close = True
-            excel_writer = pd.ExcelWriter(excel_writer)
-
-        # format and write the codelist
         codelist = (
             pd.DataFrame.from_dict(self.mapping, orient="index")
             .reset_index()
             .rename(columns={"index": self.name})
             .drop(columns="file")
         )
+        if sorted:
+            codelist.sort_values(by=self.name, inplace=True)
         codelist.rename(
```

Comment:
Is there a specific reason why we need to capitalize the columns? I was just looking at the unit test and was initially confused why it turned "bool" into "Bool".

Reply:
This is a legacy/convenience design decision because the Excel templates used to be capitalized but in the yaml files, we went the more pythonic way of all-lowercase...

Reply:
Ah right, let's not break convention then.